src/boot/efi/console.h

```diff
@@ -9,12 +9,22 @@
 #define KEYCHAR(k) ((k) & 0xffff)
 #define CHAR_CTRL(c) ((c) - 'a' + 1)
 
-enum console_mode_change_type {
-        CONSOLE_MODE_KEEP = 0,
-        CONSOLE_MODE_SET,
+enum {
+        /* Console mode is a INT32 in EFI. We use INT64 to make room for our special values. */
+        CONSOLE_MODE_RANGE_MIN = 0LL,
```

Comment:
the LL suffix here has no meaning. The type of the enum is not affected by it, i.e. sizeof(CONSOLE_MODE_RANGE_MIN) doesn't change if you add type suffixes to the rhs, hence i'd just drop them here.